jenkins-slaves/templates/jenkins-slave-generic-template.yml

```diff
@@ -0,0 +1,78 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: "${NAME}"
+  annotations:
+    openshift.io/display-name: Generic Build Pod
+    description: "${NAME} build pod template pre-configured to use a jenkins slave in the
+      same project/namespace"
+objects:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      build: "${NAME}"
+      role: jenkins-slave
+    name: "${NAME}"
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    labels:
+      build: "${NAME}"
+      type: image
+    name: "${NAME}"
+  spec:
+    nodeSelector:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: "${NAME}:latest"
```

Comment:
I think this should be `name: "${NAME}:${SLAVE_IMAGE_TAG}"`
Otherwise can you remove the `SLAVE_IMAGE_TAG` parameter from the template?